src/apps/config/leader.lua

```diff
@@ -27,7 +27,7 @@ Leader = {
       -- Could relax this requirement.
       initial_configuration = {required=true},
       schema_name = {required=true},
-      follower_pids = {required=true},
+      worker_start_fn = {required=true},
```

Comment:
What do you think about replacing this with the Lua code that would be run in the follower?  That way the leader handlers the "worker.start" bit.